icdiff

```diff
@@ -487,6 +499,9 @@ def create_option_parser():
                       action="store_true",
                       help="show the whole file instead of just changed "
                       "lines and context")
+    parser.add_option("--strip-trailing-cr", default=False,
+                      action="store_true",
+                      help="ignore windows newline carriage return")
```

Comment:
This text here should match the text you added in the readme.  Either text is fine, but let's make it consistent.